perllib/FixMyStreet/DB/Result/Body.pm

```diff
@@ -224,6 +224,17 @@ has a council_name, use it in preference to the body name.
 
 sub cobrand_name {
     my $self = shift;
+
+    # Because TfL covers all the boroughs in London, get_cobrand_handler
+    # may return another London cobrand if it is listed before tfl in
+    # ALLOWED_COBRANDS, because one of this body's area_ids will also
```

Comment:
Could we just mandate that 'tfl' must always come before any other London cobrands in that list? :)

Reply:
I've got a vague feeling that doing that caused another issue, but in my testing now I can't see anything wrong.

I guess it's a toss-up between including cobrand-specific code in the core modules and introducing a non-obvious requirement for `ALLOWED_COBRANDS` to be ordered. I suppose for the purposes of our deployments enforcing an ordering would be OK.

Reply:
Actually on the staging site now I'm seeing "reported to Bexley or Bexley" in the HTML. It's because the Cobrand.pm body_handler check does $areas->{id} which won't work for TfL when id is an array. So it wasn't to do with cobrand order, it would never have returned TfL if I've understood it right. So sorry, perhaps easiest just to put this bit back.